parlai/tasks/opensubtitles/agents.py

```diff
@@ -58,6 +58,16 @@ def setup_data(self, path):
             cnt += 1
 
 
+class Task10kTeacher(HalfTeacher):
+    """This version of opensubtitles only includes 10,000 dialogs."""
+    def setup_data(self, path):
+        cnt = 0
+        for entry, new in super().setup_data(path):
+            if cnt < 10000:
+                yield entry, new
```

Comment:
this data isn't random though, right?

Reply:
right just the first 10k (from one perspective, no reversed perspectives)